internal/providers/pke/pkeworkflow/create_master_activity.go

```diff
@@ -74,7 +74,7 @@ func (a *CreateMasterActivity) Execute(ctx context.Context, input CreateMasterAc
 
 	awsCluster, ok := cluster.(AWSCluster)
 	if !ok {
-		return "", errors.New(fmt.Sprintf("can't create VPC for cluster type %t", cluster))
+		return "", errors.Errorf("can't cast to AWS cluster %t", cluster)
```

Comment:
nit: avoid apostrophes in error messages

Reply:
thanks, fixed